src/fmu/dataio/_models/fmu_results/fields.py

```diff
@@ -680,6 +691,11 @@ class FMU(FMUBase):
     """The ``fmu.realization`` block contains information about the realization this
     data object belongs to. See :class:`Realization`."""
 
+    element: Element | None = Field(default=None)
+    """The ``fmu.element`` block identifies data objects representing the same entity
+    within a case. Note, for objects exported in context ``case`` this field will
+    be empty. See :class:`Ensemble`."""
```

Comment:
Maybe we would like to populate this for objects exported in context `case` also? If so could we just make a random UUID for those?

Reply:
Yes, that is an option, but we could wait it out a bit as well and add it if we see the need for it at a later stage? (I.e. yes, maybe, but does it have to be now?)